### PR TITLE
keystone v3 backport for tex

### PR DESCRIPTION
**Pull request by rhafer**
Depends on: crowbar/barclamp-keystone#282

### CODE REVIEW

File: chef/data_bags/crowbar/bc-template-swift.schema [hunk 1]
```diff
@@ -155,7 +155,8 @@
             "admin_ip_expr": { "type": "str", "required": true},
             "storage_ip_expr": { "type": "str", "required": true},
             "public_ip_expr": { "type": "str", "required": true},
-            "disk_zone_assign_expr": { "type": "str", "required": false}                        
+            "disk_zone_assign_expr": { "type": "str", "required": false},
+            "max_header_size": { "type": "int", "required": true },
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/bc-template-swift.schema]: syntax error here..

File: chef/data_bags/crowbar/migrate/swift/011_add_max_header_size.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def upgrade ta, td, a, d
+  a['max_header_size'] = 16384
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/migrate/swift/011_add_max_header_size.rb]: ta['max_header_size']

**Comment by vuntz** [comment]
+1.

**Comment by rsalevsky** [comment]
+1

**Pull request merged**